.github/workflows/docker.yml

```diff
@@ -18,10 +18,9 @@ jobs:
         uses: actions/checkout@master
         with:
           fetch-depth: 0
-      - name: Configure Git
-        run: |
-          git config --unset "http.https://github.com/.extraheader"
-          git fetch origin master
+          persist-credentials: false
+      - name: Fetch origin/master from Git
+        run: git fetch origin master
```

Comment:
Why are `actions/checkout` and `git fetch origin master` both needed?

Reply:
I don't believe it fetches branch refs, even with `fetch-depth: 0`

Reply:
In that case I'd suggest doing something like this: https://github.com/actions/checkout#Fetch-all-branches

Reply:
Why? We don't need all branches, just `origin/master`. I'm addressing this specific error:
```console
$ docker run -it --rm docker.pkg.github.com/homebrew/brew/ubuntu20.04:2.2.15
$ brew update
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
```

Reply:
`+refs/heads/#{ENV["GITHUB_BASE_REF"]}` is the refspec you want. It would be nice to combine that with the normal fetch for actions/checkout to avoid running two `git fetch`es.

Reply:
`git fetch origin master` takes 1.36s, so it doesn't seem worth optimizing to me.

`actions/checkout` does not currently support fetching multiple references. There's an open issue. https://github.com/actions/checkout/issues/214